docs/reference/cli.rst

```diff
@@ -138,6 +138,17 @@ Optional command flags:
   searching for other candidates by using the ``--search-id SEARCH_ID`` option.
   Multiple IDs can be specified by simply repeating the option several times.
 
+* You can supply ``--set-field`` options with ``field=value`` pairs to assign to
+  those fields the specified values on import, in addition to such field/value
+  pairs defined in the ``importer.set_fields`` dictionary in the configuration
+  file. Make sure to use an option per field/value pair, like so:
+  ::
```

Comment:
In ReST, if you just use :: at the end of your paragraph, you'll get a literal : there *and* mark the following indented block as code.